src/java/main/org/apache/zookeeper/server/ZooKeeperServer.java

```diff
@@ -270,21 +273,21 @@ public void setZKDatabase(ZKDatabase zkDb) {
      */
     public void loadData() throws IOException, InterruptedException {
         /*
-         * When a new leader starts executing Leader#lead, it 
+         * When a new leader starts executing Leader#lead, it
```

Comment:
Zookeeper policy is not to reformat existing quirks. Let's follow their rules in case we try to upstream.